### PR TITLE
New view

### PR DESCRIPTION
**Pull request by fabio-looker**
Upgraded block to New LookML and added a view/explore for recent plan steps. Also persisted certain system views to PDTs so that they can be joined in redshift

### CODE REVIEW

File: redshift_admin.view.lkml [hunk 1]
```diff
@@ -335,3 +355,91 @@ view: data_loads {
...
+
+view: recent_plan_steps {
+  #Recent is last 1000 queries
+  derived_table: {
+    # Insert into PDT because redshift won't allow joining certain system tables/views onto others (presumably because they are located only on the leader node)
+    persist_for: "1 hour"
+    sql:
+        SELECT
+        query, nodeid,
+        substring(regexp_substr(plannode, 'XN ([A-Z][a-z]+ ?)+'),3) as oper,
```
Comment (githoov) [inline redshift_admin.view.lkml:367]: I see two metacharacters next to one another: `+ ?`. Is Redshift picking up the space between them as an implicit `\\s`? Perhaps I'm missing something.

File: redshift_admin.view.lkml [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  }
+
+  dimension: rows {
+    type: number
+    sql: ${TABLE}.rows ;;
+  }
+
+  dimension: table_stem {
+    sql: case
+        when (${table} ~ '(lr|lc)\\$[a-zA-Z0-9]+_.*')
```
Comment (githoov) [inline redshift_admin.view.lkml:209]: Does Looker still use the `lc` prefix? I feel like this may have changed and I never updated this code.

**Review by githoov — COMMENTED** [review]
Looks good! I like the new derived table. Very cool!

File: redshift_admin.view.lkml (hunk 2 above)
Comment (fabio-looker) [reply redshift_admin.view.lkml:209]: Not sure on this one. For what it's worth, I didn't touch it other than running it through the converter

File: redshift_admin.view.lkml (hunk 1 above)
Comment (fabio-looker) [reply redshift_admin.view.lkml:367]: Perhaps I am being too literal with my regex construction but I think I was going for matching "WordWord" or "Word Word", but not "Word word". (I did intend for the space character in the expression is treated as a character to match... is that not the case?)

Comment (githoov) [reply redshift_admin.view.lkml:367]: Makes sense!

**Comment by githoov** [comment]
@DillonMorrison

File: redshift_admin.view.lkml (hunk 2 above)
Comment (IanAtLooker) [reply redshift_admin.view.lkml:209]: LC is still used by building tables, once the build is complete we drop the LR table and rename the LC table to LR

Comment (githoov) [reply redshift_admin.view.lkml:209]: 👍

**Comment by fabio-looker** [comment]
Does anyone approve this stuff, or are we supposed to merge our own pull requests?

**Comment by githoov** [comment]
@fabio-looker Segah and I used to merge changes; however, they took away all sorts of fun permissions when I left Looker. I'd say your best bet is @IanAtLooker.

**Pull request merged**